src/db/clients/postgresql.js

```diff
@@ -23,31 +23,33 @@ export default function(server, database) {
     const client = new Client(dbConfig);
 
     debug('connecting');
-    client.connect(err => {
+    client.connect(async err => {
       if (err) {
         client.end();
         return reject(err);
       }
 
       debug('connected');
+      const defaultSchema = await getSchema(client);
+
       resolve({
         wrapIdentifier,
         disconnect: () => disconnect(client),
-        listTables: () => listTables(client),
-        listViews: () => listViews(client),
-        listRoutines: () => listRoutines(client),
-        listTableColumns: (db, table) => listTableColumns(client, db, table),
-        listTableTriggers: (table) => listTableTriggers(client, table),
+        listTables: (_, schema = defaultSchema) => listTables(client, schema),
```

Comment:
Yeah, I know the convention 😄 . I just thought the eslint would not complain about the not used param because there is a parameter after that. But now I realized since the second parameter has a default param, makes sense eslint complain about the first parameter not been used.
Anyway, moving the `defaultSchema` to the function definition will not require use those underscores.
By the function definition I mean this: 
```js
export function listTables(client, schema = defaultSchema) {
```